src/app/CommandHandler.h

```diff
@@ -426,10 +436,15 @@ class CommandHandler : public Messaging::ExchangeDelegate
 
     bool mSentStatusResponse = false;
 
-    State mState       = State::Idle;
-    bool mGroupRequest = false;
+    State mState = State::Idle;
+
     chip::System::PacketBufferTLVWriter mCommandMessageWriter;
     TLV::TLVWriter mBackupWriter;
+    State mBackupState;
+    CommandRefLookupTable mCommandRefLookupTable;
```

Comment:
The need for this table is so that when an async command that is holding onto `CommandHandler::Handle` goes to send a response that we know what the `CommandRef` is to append to the response. Without this we cannot know for certain which Reference we need to attach for async command